src/scripts/main.js

```diff
@@ -1 +1,87 @@
-'use strict';
+const body = document.querySelector('body');
+
+function createNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type;
+  notification.innerText = message;
+  body.insertAdjacentElement('afterbegin', notification);
+}
+
+function firstPromise() {
+  return new Promise((resolve, reject) => {
+    let isClicked = false;
```

Comment:
You don't have to keep track of isClicked just send notification right away after event is triggered.